addons/spreadsheet/static/src/chart/plugins/odoo_chart_core_view_plugin.js

```diff
@@ -182,6 +173,8 @@ export class OdooChartUIPlugin extends OdooUIPlugin {
         const definition = this.getters.getChart(chartId).getDefinitionForDataSource();
         const dataSourceId = this._getOdooChartDataSourceId(chartId);
         this.charts[dataSourceId] = new ChartDataSource(this.custom, definition);
+        this._addDomain(chartId);
```

Comment:
https://www.odoo.com/odoo/project/2328/tasks/4965683 to fix this in stable and add a test